<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cvnirc-qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="LogBuffer" name="logBufferProto" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>160</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabMain">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="LogBuffer" name="logBufferMain" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelUserInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>User input</string>
        </property>
        <property name="buddy">
         <cstring>lineEditUserInput</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditUserInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonUserInput">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="action_Reconnect"/>
    <addaction name="action_Disconnect"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionLocalOnlineHelp"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionFocusUserInput"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Quit"/>
   <addaction name="action_Connect"/>
   <addaction name="action_Reconnect"/>
   <addaction name="action_Disconnect"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="icon">
    <iconset theme="application-exit" resource="cvnirc-gui.qrc">
     <normaloff>:/icons/32px-quit.png</normaloff>:/icons/32px-quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Connect">
   <property name="icon">
    <iconset resource="cvnirc-gui.qrc">
     <normaloff>:/icons/32px-connect.png</normaloff>:/icons/32px-connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="action_Reconnect">
   <property name="icon">
    <iconset resource="cvnirc-gui.qrc">
     <normaloff>:/icons/32px-reconnect.png</normaloff>:/icons/32px-reconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reconnect</string>
   </property>
  </action>
  <action name="action_Disconnect">
   <property name="icon">
    <iconset resource="cvnirc-gui.qrc">
     <normaloff>:/icons/32px-disconnect.png</normaloff>:/icons/32px-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionLocalOnlineHelp">
   <property name="text">
    <string>&amp;Local online help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionFocusUserInput">
   <property name="text">
    <string>Focus user &amp;input</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LogBuffer</class>
   <extends>QWidget</extends>
   <header>logbuffer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cvnirc-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEditUserInput</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonUserInput</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
